assets/schema_input.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This is for the manifest file that the pipeline consumes, correct? So not redundant with the nf-core schema file?

Reply:
Yes, it checks that the sample sheet contains a column called 'manifest'.  I included it as the lint check was throwing an error without it.  It's redundant as the first process, `check_inputs` checks the sample sheet more extensively.